helm-chart/README.rst

```diff
@@ -56,3 +56,43 @@ To run tests on the deployment, use
 .. code-block:: console
 
     $ helm test --cleanup renku
+
+
+Upgrading
+---------
+Most information related to upgrading from one chart version to another is covered
+in the `values changelog file<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_.
+For upgrades that require some steps other than modifying the values files to be
+executed, we add some instructions here.
+
+Upgrading to 0.8.0
+******************
+This upgrade comes with an upgrade of the keycloak chart from `4.10.2` to `9.8.1`! For
+details on this upgrade check the dedicated section in the
+`the keycloak chart docs<https://github.com/codecentric/helm-charts/tree/master/charts/keycloak#upgrading>`_
+and the `keycloak docs<https://www.keycloak.org/docs/latest/upgrading/>`_.
+
+- Before starting, make sure to check out `the values changelog for this upgrade<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md#upgrading-to-renku-080-includes-breaking-changes>`_
+  and update your values file accordingly.
+
+- The upgrade of keycloak will perform an irreversible database migration. It is therefore recommended
+  to back up your postgres volume before performing this upgrade.
+
+- If they have not been explicitly defined (and thus autocreated through a helm hook), make sure to get the value
+  of get the `keycloak-postgres-password` from the `renku-keycloak-postgres` secret and add it to your values file
+  as `global.keycloak.postgresPassword.value`. Do the same for `keycloak-password` from the `keycloak-password-secret`
+  and add it as `global.keycloak.postgres.value`.
+
+- Delete the two secrets which need to be recreated as well as the keycloak StatefulSet:
+.. code-block:: bash
+
+   kubectl delete secrets -n <namespace> keycloak-password-secret renku-keycloak-postgres
+   kubectl delete statefulsets.apps -n <namespace> <release-name>-keyclo
```

Comment:
```suggestion
   kubectl delete statefulsets.apps -n <namespace> <release-name>-key
```

Reply:
oops - I've seen so many 7-letter namespaces (andreas, ci-1234) that I've come to believe that `<release-name>-keyclo` was generic...

Reply:
fixed by querying the actual name first.